common/Makefile.common.mk

```diff
@@ -107,13 +104,6 @@ update-common:
 	@cp -a $(TMP)/common-files/files/* $(shell pwd)
 	@rm -fr $(TMP)/common-files
 
-update-common-protos:
```

Comment:
I donot know why the proto is used in main repo. Can you clarify

Reply:
Makefile.common.mk does many many things, one of them is update-common-protos. 

If you make this change, when `update-common` is ran (which is done automatically) it will revert this change anyways